build/yaml/samples/nsx_v1alpha1_subnet.yaml

```diff
@@ -15,3 +15,25 @@ spec:
   ipv4SubnetSize: 64
   subnetDHCPConfig:
     mode: DHCPServer
+---
+# Subnet CR sample for assigned VLAN extension Subnet
+apiVersion: crd.nsx.vmware.com/v1alpha1
+kind: Subnet
+metadata:
+  name: vlan-subnet
+  annotations:
+    nsx.vmware.com/associatedResource: project1:vpc1:vlan-subnet
```

Comment:
this example is not consistent with code

Reply:
changed.thanks.